common.css

```diff
@@ -0,0 +1,141 @@
+body
+{
+    background-color: var(--main-background-color);
+    box-shadow: 0 4px 12px 0 rgb(0 0 0 / 5%);
+    font-family: Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif;
+}
+a > .fa 
+{
+    font-family: "Font Awesome 5 free";
+    background-color: var(--component-black-color);
+}
+.icon-ecomm > .material-icons
+{
+  font-size: 1.5rem;
+}
+.material-icons
+{
+    color: var(--component-black-color);
+}
+.wrapper{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    gap: 2rem;
+    height: 100vh;
+    box-shadow: 0 4px 12px 0 rgb(0 0 0 / 15%);
+}
+.header-outer {
+    display: flex;
+    align-items: center;
+    position: sticky;
+    top: -2px;
+    height: 4.125rem;
+    width: 100%;
+    background-color: var(--main-background-color);
+    z-index: 999;
```

Comment:
We shouldn't put this kind of random value of z-index. Instead of it, we can set variables for it. Other than that everything is great. Go ahead and merge the branch.